packages/muelu/research/regionMG/test/structured/CMakeLists.txt

```diff
@@ -144,6 +145,7 @@ IF (${PACKAGE_NAME}_ENABLE_Tpetra AND ${PACKAGE_NAME}_ENABLE_Amesos2)
     COMM serial mpi
```

Comment:
Wouldn't it be enough to remove `serial`?

Reply:
Is that how this works? Because that does seem like it would be better.

Reply:
Yes, in combination with `NUM_MPI_PROCS 4` this should do the trick

Reply:
I will test that, then change this PR to be that if it works.

Reply:
I did it.